### ISSUE TITLE
Support Tesseract3.03~

### ISSUE THREAD

**Issue by otiai10** [otiai10/gosseract#25]
As is (almost vacant)
```go
package gosseract

type tesseract0303 struct {
	version string
}

func (t tesseract0303) Version() string {
	return t.version
}
func (t tesseract0303) Execute(args []string) (res string, e error) {
	res = "tesseract0303"
	return
}
```

**Comment by TomNomNom** [comment]
I've had a go at adding 3.03 support in TomNomNom/gosseract@59d79e34cd30897897121ff56f345b2f94b1cb7b.

The tesseract changelog doesn't indicate any breaking changes between 3.02 and 3.03 that I can see; so it's pretty much a copy and paste job.

I did find that 3.03 on my machine didn't pass the tests that use whitelists; it was interpreting some of the ':' characters in the images as 'I' characters. Seeing as it shouldn't be the concern of the gosseract tests how well tessertact itself works, I've tried to make the tests less brittle by stripping whitespace and whitelisting only characters that actually exist in the image.

I'll issue a pull request shortly.
